plugin.go

```diff
@@ -135,14 +137,15 @@ func (p Plugin) cloneTarget() error {
 func (p Plugin) rsyncPages() error {
 	cmd := exec.Command(
 		"rsync",
+		"-av",
 		"--delete",
 		"--exclude",
 		".git",
 		"--exclude",
 		"CNAME",
 		"-r",
-		p.Config.PagesDirectory,
-		p.Config.TemporaryBase,
+		strings.TrimRight(p.Config.PagesDirectory, "/")+"/",
+		path.Join(p.Config.WorkDirectory, p.Config.TargetDirectory),
```

Comment:
Did you mix something here? Before it used `p.Config.TemporaryBase` which is not used at all anymore?

Reply:
Just sync the all files in `p.Config.PagesDirectory + "/"` to target clone base not include the `p.Config.PagesDirectory`.

origin:

sync `p.Config.PagesDirectory` to `p.Config.TemporaryBase`

after:

sync `p.Config.PagesDirectory + "/"` to `p.Config.WorkDirectory`

Reply:
But the repo gets cloned to the temporary base so far?!

Reply:
Yes, the default behavior is sync all files in `PagesDirectory` include the self folder to `TemporaryBase`.

Change to sync all files in `PagesDirectory` **not** include the self folder. 

So you can define the `TargetFolder` to sync all files in `PagesDirectory` into `TargetFolder` folder base on `WorkDirectory`.

Reply:
example:

https://github.com/golangtw/gophercon2020/blob/7e37a6c300bbfca658e030544609b6fbbd319a67/.drone.yml#L23

```yaml
- name: publish
  image: appleboy/gh-pages
  pull: always
  settings:
    username:
      from_secret: username
    password:
      from_secret: password
    pages_directory: dist
    target_branch: master
    target_directory: 2020
    remote_url: https://github.com/golangtw/gophercon.github.io.git
  when:
    branch:
    - master
    event:
    - push
```